src/Features/SupportConsoleCommands/Tests/FileManipulationCommandParserUnitTest.php

```diff
@@ -3,13 +3,12 @@
 namespace Livewire\Features\SupportConsoleCommands\Tests;
 
 use Livewire\Features\SupportConsoleCommands\Commands\ComponentParser;
+use PHPUnit\Framework\Attributes\{Test, DataProvider};
 
 class FileManipulationCommandParserUnitTest extends \Tests\TestCase
 {
-    /**
-     * @test
-     * @dataProvider classPathProvider
-     */
+    #[Test]
+    #[DataProvider('classPathProvider')]
```

Comment:
Using PHPUnit Annotation will trigger a deprecation warning in PHPUnit 11, it's best to migrate to PHPUnit Attributes